ts/src/editor.ts

```diff
@@ -494,20 +490,20 @@ allowedTagsExtended["SPAN"] = filterExternalSpan;
 Object.assign(allowedTagsExtended, allowedTagsBasic);
 
 // filtering from another field
-var filterInternalNode = function(node) {
+const filterInternalNode = function(node) {
```

Comment:
It's probably better to use let for global functions, as there may be add-ons out there that are rebinding the functions to behave differently.

Reply:
Thanks, done